build_tools/benchmarks/comparisons/common/benchmark_runner.py

```diff
@@ -53,8 +54,8 @@ def run_command(benchmark_command: BenchmarkCommand) -> list[float]:
   stdout_data, _ = benchmark_process.communicate()
 
   if benchmark_process.returncode != 0:
-    print("Warning! Benchmark command failed with return code: " + str(
-        benchmark_process.returncode))
+    print("Warning! Benchmark command failed with return code: " +
+          str(benchmark_process.returncode))
```

Comment:
This could be an f-string, which is what we generally use in our scripts for string formatting

```suggestion
    print(f"Warning! Benchmark command failed with return code:"
          f" {benchmark_process.returncode}")
```